### PR TITLE
Don't send me to null island, please

### PR DESCRIPTION
**Pull request by thebentern**
@garthvh I _think_ this should fix the null island brigade

### CODE REVIEW

File: src/modules/PositionModule.cpp [hunk 1]
```diff
@@ -133,20 +133,20 @@ int32_t PositionModule::runOnce()
...
-
-            DEBUG_MSG("Sending pos@%x:6 to mesh (wantReplies=%d)\n", node->position.pos_timestamp, requestReplies);
-
-            sendOurPosition(NODENUM_BROADCAST, requestReplies);
+                DEBUG_MSG("Sending pos@%x:6 to mesh (wantReplies=%d)\n", node->position.pos_timestamp, requestReplies);
 
+                sendOurPosition(NODENUM_BROADCAST, requestReplies);
+            }
         } else {
             DEBUG_MSG("Channel utilization is >50 percent. Skipping this opportunity to send.\n");
```
Comment (caveman99) [inline src/modules/PositionModule.cpp:151]: Completely unrelated to the PR, why does it say 50% down here when it's 40% up there?

**Review by caveman99 — COMMENTED** [review]
looks like a good catch - however there are similar checks all over the GPS code. Our 'Null Island' on T-Echo was not always 0.00/0.00 but 0.something/0.something

File: src/modules/PositionModule.cpp (hunk 1 above)
Comment (thebentern) [reply src/modules/PositionModule.cpp:151]: lol, that's a nice misleading typo

**Comment by thebentern** [comment]
> looks like a good catch - however there are similar checks all over the GPS code. Our 'Null Island' on T-Echo was not always 0.00/0.00 but 0.something/0.something

Garth was reporting zeros on non-smart broadcast position packets, but I think that was for mostly for non T-Echo devices. I'm gonna add this guard for now just to be consistent, since we have the same on smart broadcast. 
Do you have some example values for what to look for in the T-Echos? Is it close enough to do some fuzzy truncation logic? I can do a follow up ticket to address it.

**Comment by caveman99** [comment]
I haven't seen the specific T-Echo problems since i swapped out the math in TinyGPS. Fingers crossed. Just adding the note to keep your eyes peeled if this is happening again.

File: src/modules/PositionModule.cpp (hunk 1 above)
Comment (mc-hamster) [reply src/modules/PositionModule.cpp]: Should be 40%

Comment (thebentern) [reply src/modules/PositionModule.cpp]: Typo fix made it into the merge :-)

**Comment by geeksville** [comment]
This pull request has been mentioned on **Meshtastic**. There might be relevant details there:

https://meshtastic.discourse.group/t/meshtastic-1-3-29-public-preview/6241/1

**Pull request merged**